install_sct

```diff
@@ -552,45 +588,38 @@ if [ "$ACTUAL_PYTHON" != "$EXPECTED_PYTHON"  ]; then
 fi
 
 
-# Install Python dependencies
+## Install the spinalcordtoolbox into the Conda venv
 print info "Installing Python dependencies..."
 # Check if a frozen version of the requirements exist (for release only)
 if [[ -f "requirements-freeze.txt" ]]; then
   print info "Using requirements-freeze.txt (release installation)"
-  export REQUIREMENTS_FILE="requirements-freeze.txt"
+  REQUIREMENTS_FILE="requirements-freeze.txt"
 else
   # Not a package
   print info "Using requirements.txt (git installation)"
-  export REQUIREMENTS_FILE="requirements.txt"
+  REQUIREMENTS_FILE="requirements.txt"
 fi
-pip install -r $REQUIREMENTS_FILE &&
+(
+  pip install -r "$REQUIREMENTS_FILE" &&
   # `tensorflow-tensorboard` is installed by `tensorflow==1.5.0` but is not needed,
   # and conflicts with the other installation of `tensorboard`. So, we uninstall here.
   # This is hacky, and should be removed as soon as we stop using `tensorflow==1.5.0`.
   # See https://github.com/neuropoly/spinalcordtoolbox/issues/3035 for more info.
-  yes | pip uninstall tensorflow-tensorboard &&
-  yes | pip uninstall tensorboard &&
-  pip install tensorboard
-if [[ $? != 0 ]]; then
+  pip uninstall -y tensorflow-tensorboard &&
+  pip uninstall -y tensorboard &&
+  pip install tensorboard &&
+  # install sct itself
+  print info "Installing spinalcordtoolbox..." &&
+  pip install -e .
```

Comment:
It's redundant to run `pip install $SCT_DIR`  and also `pip install -r requirements.txt`, but that's how it was when I found it.

Also, `pip install -e`? That's really more meant for developers, not for installations. But we can deal with that in #1526. pip-only install! right @Drulex ?